packages/api/src/hooks/p2p/useAdvertiserRelations.ts

```diff
@@ -0,0 +1,20 @@
+import useQuery from '../../useQuery';
+import useAuthorize from '../useAuthorize';
+
+/** This hook returns favourite and blocked advertisers of the current user. */
+const useAdvertiserRelations = () => {
+    const { isSuccess } = useAuthorize();
+    const { data, ...rest } = useQuery('p2p_advertiser_relations', { options: { enabled: isSuccess } });
+    const advertiser_relations = data?.p2p_advertiser_relations;
```

Comment:
```suggestion
```

Reply:
I don't want to repeat this multiple times that's why I created a variable for it